lib/process.js

```diff
@@ -28,13 +28,14 @@ function processScript (scriptPart) {
   return compileBabel(scriptPart.content)
 }
 
-function processStyle (stylePart) {
+function processStyle (stylePart, path) {
   if (!stylePart) return {}
 
   let cssCode = stylePart.content
   if (/^styl|stylus$/.test(stylePart.lang)) {
+    const vuePath = join(path, '..')
```

Comment:
Can you use process.cwd for the path? Like in vueify—https://github.com/vuejs/vueify/blob/master/lib/compilers/stylus.js#L14